### PR TITLE
Token Authentication, PHP samples

### PR DESCRIPTION
**Pull request by bartlomiejwereszczynski**

### CODE REVIEW

File: public/ably.php [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<?php
+require_once __DIR__ . '/../vendor/autoload.php';
+
+throw new \RuntimeException('Insert your API key'); // REMOVE THIS
+$ably = new \Ably\AblyRest(["key" => "{{ApiKey}}"]);
```
Comment (mattheworiordan) [inline public/ably.php:5]: Out of interest why do you use this format here `["key" => "{{ApiKey}}"]` and in the other demo you passed only a key string, and in the tutorial itself you use a string too, see https://github.com/ably/docs/pull/164/commits/d0084781cf6ae58248c8b6a2b6f64a5b472d23f8#diff-a551a3cd350dc478774a772f537a027fR74

File: public/auth.php [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<?php
+require_once 'ably.php';
+header('Content-Type: application/json');
+echo json_encode($ably->auth->createTokenRequest()->toArray());
```
Comment (mattheworiordan) [inline public/auth.php]: Slightly inconsistent with the tutorial i.e. useful comment at https://github.com/ably/docs/pull/164/commits/d0084781cf6ae58248c8b6a2b6f64a5b472d23f8#diff-a551a3cd350dc478774a772f537a027fR248 is missing

File: public/auth.php [hunk 3]
```diff
@@ -1,4 +1,23 @@
...
 require_once 'ably.php';
+// Check if the user is logged in
+if (isset($_COOKIE['username'])) {
+    // Issue a token request with pub & sub permissions on all channels +
+    // configure the token with an identity
+    $tokenParams = [
+        'capability' => [
+            '*' => ['publish', 'subscribe']
+        ],
+        'clientId'  => $_COOKIE['username']
```
Comment (mattheworiordan) [inline public/auth.php:11]: minor comment but extra space before `=>`

**Review by mattheworiordan — CHANGES_REQUESTED** [review]
Looks good, few comments to address

File: public/ably.php (hunk 1 above)
Comment (bartlomiejwereszczynski) [reply public/ably.php:5]: tutorial for ruby and node used similar syntax, and in docs i've copied that block from "history" tutorial, i'll update it to be correct with code sample

File: public/auth.php (hunk 2 above)
Comment (bartlomiejwereszczynski) [reply public/auth.php]: ok, i''l add that comment in code

File: public/auth.php (hunk 3 above)
Comment (bartlomiejwereszczynski) [reply public/auth.php:11]: formatting error, i was aiming to align both `=>` in same column, i'll fix it in second version

**Comment by bartlomiejwereszczynski** [comment]
recreated version is available here: https://github.com/ably/tutorials/pull/9

**Pull request closed without merging**